graphql_api/types.py

```diff
@@ -487,6 +488,22 @@ class Meta:
     id = graphene.ID(source="id", required=True)
 
 
+class LabelingAccessLink(SQLAlchemyObjectType):
+    class Meta:
+        model = models.LabelingAccessLink
+        interfaces = (Node,)
+
+    id = graphene.ID(source="id", required=True)
+    name = graphene.String()
+
+    def resolve_name(self, info):
+        if self.link_type == enums.LinkTypes.HEURISTIC.value:
+            return information_source.get(self.project_id, self.heuristic_id).name
+        elif self.link_type == enums.LinkTypes.DATA_SLICE.value:
+            return data_slice.get(self.project_id, self.data_slice_id).name
+        return "Unknown type"
```

Comment:
maybe rather raise an exception?

Reply:
i'd rather not since this "only" a name so having the name "unknown type" wouldn't break the application